crates/swift-bridge-ir/src/parse/parse_extern_mod/argument_attributes.rs

```diff
@@ -0,0 +1,85 @@
+use proc_macro2::Ident;
+use syn::parse::{Parse, ParseStream};
+use syn::{LitStr, Token};
+
+#[derive(Default)]
+pub(super) struct ArgumentAttributes {
+    /// LitStr: argument_name
+    pub label: Option<LitStr>,
+}
+
+enum ArgumentAttr {
+    /// LitStr: argument_name
+    ArgumentLabel(LitStr),
+}
+
+impl Parse for ArgumentAttributes {
+    fn parse(input: ParseStream) -> syn::Result<Self> {
+        let mut attributes = ArgumentAttributes::default();
+        let punctuated =
+            syn::punctuated::Punctuated::<ArgumentAttr, syn::Token![,]>::parse_terminated(input)?;
+        for attr in punctuated.into_iter() {
+            match attr {
+                ArgumentAttr::ArgumentLabel(label) => {
+                    attributes.label = Some(label);
+                }
+            }
+        }
+        Ok(attributes)
+    }
+}
+
+impl Parse for ArgumentAttr {
+    fn parse(input: ParseStream) -> syn::Result<Self> {
+        let key: Ident = input.parse()?;
+        let attribute = match key.to_string().as_str() {
+            "label" => {
+                input.parse::<Token![=]>()?;
+                let value: LitStr = input.parse()?;
+                ArgumentAttr::ArgumentLabel(value)
+            }
+            _ => {
```

Comment:
We should add a UI test file for the unrecognized argument attribute error.

Similar to https://github.com/chinedufn/swift-bridge/blob/e0d3d62336cfb909d299ee1b75b89fc0c802b3b0/crates/swift-bridge-macro/tests/ui/unrecognized-function-attribute.rs#L1-L12

https://github.com/chinedufn/swift-bridge/blob/c75d6e744ac64e21145afbe4d251486f861e90cd/crates/swift-bridge-macro/tests/ui/unrecognized-function-attribute.stderr#L1-L5